### PR TITLE
Update site config for l10n teams and 1.11 info

### PR DESCRIPTION
**Pull request by zacharysarah**
/ref #10666 

This PR updates `release-1.11` with:
- v1.12 site configuration (👈 @zparnold)
- Site support and ownership for l10n teams (#10485)

### CODE REVIEW

**Comment by zacharysarah** [comment]
@zparnold 👋 Just realized there was a missed step in the 1.12 release: updating the `release-1.11:config.toml` to deprecate `release-1.11` and add info for `release-1.12`. This PR should take care of it, just surfacing for visibility.

/assign @tfogo 
See also #10668

File: config.toml [hunk 1]
```diff
@@ -102,11 +123,29 @@ description = "Production-Grade Container Orchestration"
 languageName ="English"
 # Weight used for sorting.
 weight = 1
-[languages.cn]
+
+[languages.zh]
 title = "Kubernetes"
 description = "Production-Grade Container Orchestration"
```
Comment (xiangpengzhao) [inline config.toml]: Should this description use Chinese? Same concern for below languages.

File: config.toml [hunk 2]
```diff
@@ -102,11 +123,29 @@ description = "Production-Grade Container Orchestration"
...
+weight = 3
+contentDir = "content/ko"
 
+[languages.no]
+title = "Kubernetes"
+description = "Production-Grade Container Orchestration"
+languageName ="Norsk"
+weight = 4
+contentDir = "content/no"
+
```
Comment (xiangpengzhao) [inline config.toml]: Missed `languages.ja`?

File: config.toml [hunk 3]
```diff
@@ -102,11 +123,29 @@ description = "Production-Grade Container Orchestration"
...
+contentDir = "content/zh"
 
+[languages.ko]
+title = "Kubernetes"
+description = "Production-Grade Container Orchestration"
+languageName = "한국어 Korean"
+weight = 3
+contentDir = "content/ko"
 
+[languages.no]
```
Comment (xiangpengzhao) [inline config.toml]: Don't see `content/no/OWNERS` file for this.

Comment (zacharysarah) [reply config.toml]: @xiangpengzhao It's not strictly needed, as `content/no/` is currently a test directory.

File: config.toml (hunk 2 above)
Comment (zacharysarah) [reply config.toml]: @xiangpengzhao The team hasn't indicated that they've met the minimum milestones yet. @cstoku Any updates on progress?

File: config.toml (hunk 1 above)
Comment (zacharysarah) [reply config.toml]: If the team would like to submit a PR for Chinese strings after this PR merges, that would be awesome. 👍

**Comment by zacharysarah** [comment]
@chenopis 👋 It looks like netlify is failing:
```
11:47:11 AM: ERROR 2018/10/19 18:47:11 error processing shortcode "shortcodes/deprecationwarning.html" for page "_index.html": template: shortcodes/deprecationwarning.html:7:65: executing "shortcodes/deprecationwarning.html" at <.Params.currenturl>: can't evaluate field currenturl in type interface {}
```
The value of `currentURL` matches that of `release-1.12` and `release-1.10` (#10688), which run without failing. I'm at a bit of a loss as to how to diagnose/resolve the cause here. 

Any advice?

**Comment by zacharysarah** [comment]
@chenopis Ping

File: config.toml [hunk 4]
```diff
@@ -27,49 +33,79 @@ fractions = false
...
+  "callouts",
+  "styles",
+  "custom-jekyll/tags"
+]
+js = [
+  "custom-jekyll/tags",
+  "script"
+]
+
 [[params.versions]]
```
Comment (tfogo) [inline config.toml]: In the past we haven't included links to newer versions in older docs - do we want to commit to keeping this up to date or is the idea that each version at least has a link to the next version?

Comment (zacharysarah) [reply config.toml]: @tfogo In the past, we _have_ updated older versions with newer nav info. 

If you look at the production version of [`config.toml` in `release-1.10`](https://github.com/kubernetes/website/blob/release-1.10/config.toml#L50), you can see that 1.10 was updated when 1.11 released. The updates for 1.12 were missed; that's why this PR and #10668 include updates for 1.12.

As to why we update previous versions: if you visit the [1.11 production build](https://v1-11.docs.kubernetes.io/), you can see that there's no way to navigate back to 1.12 in the version selector. This PR's [preview build](https://deploy-preview-10667--k8sio-v1-11.netlify.com/) shows 1.12 added to the version selector.

**Pull request merged**